x/mongo/driver/operation.go

```diff
@@ -188,6 +188,30 @@ func redactFinishedInformationResponse(info finishedInformation) bson.Raw {
 	return bson.Raw{}
 }
 
+// Operation Names should be sourced from the command reference documentation:
+// https://www.mongodb.com/docs/manual/reference/command/
+const (
+	AbortTransactionOp  = "abortTransaction"  // AbortTransactionOp is the name for aborting a transaction
```

Comment:
There is a requirement for a subset of these operations to be exported to avoid magic strings in the Execute method , see [here](https://github.com/mongodb/mongo-go-driver/pull/1325/files#diff-490a6369c25d838f3cbdcb7bec2c0f750b5319831509b0d7a1a53e0933245d24R1028) .

I will move these constants to the internal package.